tools/example-showcase/src/main.rs

```diff
@@ -692,7 +692,19 @@ fn parse_examples() -> Vec<Example> {
                 path: val["path"].as_str().unwrap().to_string(),
                 name: metadata["name"].as_str().unwrap().to_string(),
                 description: metadata["description"].as_str().unwrap().to_string(),
-                category: metadata["category"].as_str().unwrap().to_string(),
+                category: metadata["category"]
+                    .as_str()
+                    .unwrap()
+                    .replace(['(', ')'], "")
+                    .split_whitespace()
+                    .map(|piece| piece.to_lowercase())
+                    .fold(String::new(), |accumulator, next_piece| {
+                        if accumulator.is_empty() {
+                            next_piece
+                        } else {
+                            format!("{}-{}", accumulator, next_piece)
+                        }
+                    }),
```

Comment:
```suggestion
                    .to_lowercase()
                    .replace(['(', ')'], "")
                    .split_whitespace()
                    .collect::<Vec<_>>()
                    .join("-"),
```

Reply:
Something like https://docs.rs/slug/latest/slug/fn.slugify.html would be more robust, but this should work for the current set of categories.